config/thermofloor/heatit20a.xml

```diff
@@ -0,0 +1,225 @@
+				<Instance index="9" endpoint="8" />
+				<Instance index="10" endpoint="9" />
+				<Instance index="11" endpoint="10" />
+	</CommandClass>
+	<CommandClass id="37" name="COMMAND_CLASS_SWITCH_BINARY" version="1" request_flags="5" issecured="true" innif="true">
+				<Instance index="1" />
+				<Instance index="2" endpoint="1" />
+				<Instance index="3" endpoint="2" />
+				<Instance index="4" endpoint="3" />
+				<Instance index="5" endpoint="4" />
+				<Instance index="6" endpoint="5" />
+				<Instance index="7" endpoint="6" />
+				<Instance index="8" endpoint="7" />
+				<Instance index="9" endpoint="8" />
+				<Instance index="10" endpoint="9" />
+				<Instance index="11" endpoint="10" />
+				<Value type="bool" genre="user" instance="1" index="0" label="Switch" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="False" />
+				<Value type="bool" genre="user" instance="2" index="0" label="Switch 01" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="False" />
+				<Value type="bool" genre="user" instance="3" index="0" label="Switch 02" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="False" />
+				<Value type="bool" genre="user" instance="4" index="0" label="Switch 03" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="False" />
+				<Value type="bool" genre="user" instance="5" index="0" label="Switch 04" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="False" />
+				<Value type="bool" genre="user" instance="6" index="0" label="Switch 05" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="False" />
+				<Value type="bool" genre="user" instance="7" index="0" label="Switch 06" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="False" />
+				<Value type="bool" genre="user" instance="8" index="0" label="Switch 07" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="False" />
+				<Value type="bool" genre="user" instance="9" index="0" label="Switch 08" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="False" />
+				<Value type="bool" genre="user" instance="10" index="0" label="Switch 09" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="False" />
+				<Value type="bool" genre="user" instance="11" index="0" label="Switch 10" units="" read_only="false" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="False" />
+	</CommandClass>
+	<CommandClass id="38" name="COMMAND_CLASS_SWITCH_MULTILEVEL" version="1">
```

Comment:
This should not be required. It should be auto discovered.

Reply:
Removed

Reply:
@larstobi Did you ever get this to work? I've bought the HeatIt Z Water unit, and the 10 binary switches for the relays is not auto-discovered. Adding similar lines as above makes them appear in ozwadmin/Home Assistant, but I'm very new to all this and not sure if that's the way to do it. Without adding a line for each binary switch, only a single binary switch is reported, which toggles all 10 relays simultaneously, which is not ideal as it defeats the purpose of this device.

@Fishwaldo What can make this auto-discovery fail? Any setting on my end that can be changed? Running OpenZWave as an addon to for Home Assistant, based on the commit 66619eda6caaf2b41d07d68672efa7170f6c4722